content/platforms/kubernetes/quick-start.md

```diff
@@ -0,0 +1,164 @@
+Operator.
+
+If you are using OpenShift, see [Getting Started with OpenShift and Kubernetes]({{< relref "/platforms/kubernetes/openshift/_index.md" >}}) for
+installation information
+
+If you are using PKS, see [Getting Started with PKS (Pivotal Container Service)]({{< relref "/platforms/kubernetes/pks/_index.md" >}}) for
+installation information.
+
+Otherwise, the operator can be installed on a supported Kubernetes platform
+directly by the following procedure outlined here.
+
+## Overview of the operator
+
+The operator acts on [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRD) that describe Redis Enterprise clusters and databases. The operator itself is a deployment, backed by pods, that runs within your deployment namespace. These operator pods must run with sufficient privileges to create the Redis Enterprise cluster resources within that namespace.
+
+When installed, the following are created:
+
+ * a service account under which the operator will run,
+ * a set of Roles that define the privileges necessary for the operator to perform its tasks,
+ * a set of Role Bindings to the service account to authorize the service account to have the correct roles (see above),
+ * the CRD for a Redis Enterprise Cluster,
+ * the CRD for a Redis Enterprise Database,
+ * the operator (a deployment).
+
+ The operator currently runs within single namespace and is scoped to operate only on cluster CRD's in that namespace. This allows a multi-tenant deployment of the operator supporting different versions in different namespaces.
+
+## Installing the operator
+
+The definition and reference materials for the operator are on
+[GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs) and the
+implementation is published as a Docker container. The operator
+definitions are packaged as a single generic YAML file called
+[bundle.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/bundle.yaml).
+
+You need to ensure you pull the correct version of the bundle. The version tags
+can be found by checking the [releases on Github](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
+or via the GitHub API.
+
+The bundle for the latest release can be downloaded via `curl` using the following:
+
+```
+
+VERSION=`curl --silent https://api.github.com/repos/RedisLabs/redis-enterprise-k8s-docs/releases/latest | jq -r .tag_nam`
+curl --silent -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/$VERSION/bundle.yaml
+```
+
+or you can replace `VERSION` in the above with a specific release tag you desire.
+
+
+If you inspect this YAML file, you will find definitions for the various items
+described in the previous section.
+
+You can install the operator with a single apply command:
+
+```
+kubectl apply -f bundle.yaml -n my-namespace
```

Comment:
one way to simplify this, as we do in other guides, is to change the context to the namespace you're operating in so you won't need to specify the namespace explicitly every time. WDYT? why did you choose to use `-n`?

Reply:
It's the most generic from a cut-n-paste scenario ... if you presume a default namespace, people can't cut-n-paste a command without doing that first. consistency is better... I will add: kubectl config set-context --current --namespace=

Reply:
I have added a section on namespaces that clarifies this.